cartoframes/data/services/geocoding.py

```diff
@@ -189,11 +189,12 @@ def _hash_as_big_int(text):
 def _set_pre_summary_info(summary, output):
     logging.debug(summary)
     output['total_rows'] = sum(summary.values())
-    output['required_quota'] = sum([summary[s] for s in ['new_geocoded', 'new_nongeocoded', 'changed_geocoded', 'changed_nongeocoded']])
+    output['required_quota'] = sum(
+        [summary[s] for s in ['new_geocoded', 'new_nongeocoded', 'changed_geocoded', 'changed_nongeocoded']])
```

Comment:
This can be turned into a generator by removing the outer-most `[...]`, which will be a tad more performant:
```python
output['required_quota'] = sum(
        summary[s] for s in ['new_geocoded', 'new_nongeocoded', 'changed_geocoded', 'changed_nongeocoded'])
```

Reply:
Thanks for the tip!